builds/delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -284,10 +284,25 @@ def unDeployService(stage) {
 			} else {
 				env_key = "PROD"
 			}
-			def envBucketKey = "${env_key}${configLoader.JAZZ.S3_BUCKET_NAME_SUFFIX}"
-			sh "serverless remove --stage ${stage} --verbose  --bucket ${configLoader.AWS.S3[envBucketKey]}"
-
-			echo "Service undeployed"
+			def envBucketKey = "${env_key}${configLoader.JAZZ.S3_BUCKET_NAME_SUFFIX}"			
+			/*empty bucket before deleting it. */			
+			if(service_config['event_source_s3'] && (service_config['event_source_s3']!= null)){
+				// Empty bucket
+				def targetBucket = service_config['event_source_s3'];
+				if(env_key == "DEV"){
+					targetBucket = service_config['event_source_s3'] + "-" + stage;
```

Comment:
check with stage != "prod" for a consistency, otherwise it wont postfix for the environement "stg"

Reply:
changed condition env_key = DEV to not = PROD